app/survey_config/social_survey_config.py

```diff
@@ -32,6 +32,8 @@ def __post_init__(self):
         if not self.account_service_log_out_url:
             self.account_service_log_out_url: str = f"{self.base_url}/sign-in/logout"
 
+        self.contact_us_url: str = f"{ONS_URL}/aboutus/contactus/surveyenquiries"
```

Comment:
Now the changes have been made, it is clear that we don't need a `Welsh` config. The change is identical in both normal and Welsh config. The only thing that changes is the base url for RH and the base url for ONS. Given the paths are identical, I suggest doing this in 1 class making it easier to manage.

It then simplifies the changes to a few lines:
You need to add `language_code` to SurveyConfig.
```py
    def __post_init__(self):
        super().__post_init__()

        if self.schema:
            self.data_layer: list[dict] = [
                {
                    key: self.schema.json[key]
                    for key in ["survey_id", "title"]
                    if key in self.schema.json
                }
            ]
        
        upstream_base_url = f"{self.base_url}/{self.language_code}"
        ons_url = CY_ONS_URL if self.language_code == "cy" else ONS_URL
        
        if not self.account_service_log_out_url:
            self.account_service_log_out_url: str = f"{upstream_base_url}/start/"

        self.account_service_log_out_url: str = f"{upstream_base_url}/start"

        self.cookie_settings_url: str = f"{upstream_base_url}/cookies/"
        self.privacy_and_data_protection_url: str = (
            f"{upstream_base_url}/privacy-and-data-protection/"
        )

        self.contact_us_url: str = f"{ons_url}/aboutus/contactus/surveyenquiries"

        self.accessibility_url: str = f"{ons_url}/help/accessibility/"
        self.what_we_do_url: str = f"{ons_url}/aboutus/whatwedo/"
```

Reply:
Refactored, so we have one social survey config now, thanks!